src/ord.satyg

```diff
@@ -6,6 +6,29 @@
 
 type ordering = Lt | Gt | Eq
 
+module Ordering : sig
```

Comment:
Added eq for `ordering`. I'm a bit afraid that `Ord` and `Ordering` are confusing. Any suggestion?